browser/request_otr/request_otr_browsertest.cc

```diff
@@ -357,6 +365,7 @@ IN_PROC_BROWSER_TEST_F(RequestOTRBrowserTest,
   ASSERT_EQ(GetHistoryCount(), 0);
 }
 
+#if 0
```

Comment:
?

Reply:
re-enabled some tests in https://github.com/brave/brave-core/commit/c83ae23838b8132c40d9ce007c338abf52804ae9 but I have questions about two others (marked DISABLED_ for now)

Reply:
re-enabled remaining tests in https://github.com/brave/brave-core/pull/19965/commits/9d2cd455b79ef41c0564182c31e2755afaa1ecfd